.aws/deploy/support-task-definition.staging.json

```diff
@@ -0,0 +1,311 @@
+        {
+          "name": "DD_AGENT_MAJOR_VERSION",
+          "valueFrom": "STAGING_DD_AGENT_MAJOR_VERSION"
+        },
+        { "name": "DD_ENV", "valueFrom": "STAGING_DD_ENV" },
+        {
+          "name": "DD_LOGS_INJECTION",
+          "valueFrom": "STAGING_DD_LOGS_INJECTION"
+        },
+        { "name": "DD_SERVICE", "valueFrom": "STAGING_DD_SERVICE" },
+        { "name": "DD_TAGS", "valueFrom": "STAGING_DD_TAGS" },
+        {
+          "name": "DD_TRACE_STARTUP_LOGS",
+          "valueFrom": "STAGING_DD_TRACE_STARTUP_LOGS"
+        },
+        {
+          "name": "E2E_TEST_GH_TOKEN",
+          "valueFrom": "STAGING_E2E_TEST_GH_TOKEN"
+        },
+        { "name": "E2E_TEST_REPO", "valueFrom": "STAGING_E2E_TEST_REPO" },
+        { "name": "E2E_TEST_SECRET", "valueFrom": "STAGING_E2E_TEST_SECRET" },
+        { "name": "EFS_VOL_PATH", "valueFrom": "STAGING_EFS_VOL_PATH" },
+        {
+          "name": "ENCRYPTION_SECRET",
+          "valueFrom": "STAGING_ENCRYPTION_SECRET"
+        },
+        {
+          "name": "FF_DEPRECATE_SITE_QUEUES",
+          "valueFrom": "STAGING_FF_DEPRECATE_SITE_QUEUES"
+        },
+        { "name": "FRONTEND_URL", "valueFrom": "STAGING_FRONTEND_URL" },
+        {
+          "name": "GGS_REPAIR_FORM_KEY",
+          "valueFrom": "STAGING_GGS_REPAIR_FORM_KEY"
+        },
+        {
+          "name": "GGS_EXPERIMENTAL_TRACKING_SITES",
+          "valueFrom": "STAGING_GGS_EXPERIMENTAL_TRACKING_SITES"
+        },
+        {
+          "name": "GITHUB_BUILD_ORG_NAME",
+          "valueFrom": "STAGING_GITHUB_BUILD_ORG_NAME"
+        },
+        {
+          "name": "GITHUB_BUILD_REPO_NAME",
+          "valueFrom": "STAGING_GITHUB_BUILD_REPO_NAME"
+        },
+        { "name": "GITHUB_ORG_NAME", "valueFrom": "STAGING_GITHUB_ORG_NAME" },
+        {
+          "name": "GROWTHBOOK_CLIENT_KEY",
+          "valueFrom": "STAGING_GROWTHBOOK_CLIENT_KEY"
+        },
+        {
+          "name": "INCOMING_QUEUE_URL",
+          "valueFrom": "STAGING_INCOMING_QUEUE_URL"
+        },
+        {
+          "name": "ISOMERPAGES_REPO_PAGE_COUNT",
+          "valueFrom": "STAGING_ISOMERPAGES_REPO_PAGE_COUNT"
+        },
+        { "name": "JWT_SECRET", "valueFrom": "STAGING_JWT_SECRET" },
+        {
+          "name": "MAX_NUM_OTP_ATTEMPTS",
+          "valueFrom": "STAGING_MAX_NUM_OTP_ATTEMPTS"
+        },
+        {
+          "name": "MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS",
+          "valueFrom": "STAGING_MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS"
+        },
+        { "name": "MUTEX_TABLE_NAME", "valueFrom": "STAGING_MUTEX_TABLE_NAME" },
+        {
+          "name": "NETLIFY_ACCESS_TOKEN",
+          "valueFrom": "STAGING_NETLIFY_ACCESS_TOKEN"
+        },
+        { "name": "NODE_ENV", "valueFrom": "STAGING_NODE_ENV" },
+        { "name": "OTP_EXPIRY", "valueFrom": "STAGING_OTP_EXPIRY" },
+        { "name": "OTP_SECRET", "valueFrom": "STAGING_OTP_SECRET" },
```

Comment:
Should some of these variables removed from the support container? I assume they are not useful there? I don't have a sense of the full list of what's needed and not, but general principle is to only give the app what it needs.

Reply:
i copied it because i didn't want to sift through and manually find out what env vars are/are not used.

Reply:
Bit painful yes, but we should do it anyway for good hygiene. If the service doesn't need to know of certain secrets or values, we shouldn't pass them in.

Reply:
sounds good! you ok if i do this in a follow-up? my immediate priority now is to get functionality up and merged - will address this hygiene issue downstream